### PR TITLE
Fix email subject for "user created" mail

### PR DESCRIPTION
**Pull request by dexif**
And user link also

### CODE REVIEW

File: resources/views/emails/userCreated.blade.php [hunk 1]
```diff
@@ -3,4 +3,4 @@
 {{ $blueprint->user->username }} just joined {{ app()->url() }}.
 
 To view this new user's profile, please click on following link:
-{{ app()->url() }}/u/{{ $blueprint->user->id }}
+{{ app()->url() }}/u/{{ $blueprint->user->username }}
```
Comment (dsevillamartin) [inline resources/views/emails/userCreated.blade.php]: Why is this changed ? If the user's username is changed, the URL will no longer be valid. And their username is already present in the 3rd line.

**Review by dsevillamartin — CHANGES_REQUESTED** [review]

File: resources/views/emails/userCreated.blade.php (hunk 1 above)
Comment (dexif) [reply resources/views/emails/userCreated.blade.php]: Oh... You are right )

**Review by KyrneDev — APPROVED** [review]

**Comment by dsevillamartin** [comment]
Sorry we forgot about this PR, @dexif. 🙈
